Linear Regression/Linear_Regression.py

```diff
@@ -0,0 +1,78 @@
+"""
+Linear regression
+Arguments - alpha (by default .001)
+            epochs(by default 100)
+            plot : bool - plot the cost vs epoch graph  
+function -  fit : (x, y) - train the data
+            predict : (x) - predict the new y using previus trained model
+            this module also normalize data brfore training that increase training efficiency
+"""
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+
+class LinearRegression:
+    
+    def __init__(self):
+        self.costs = []
+        self.b = 0
+        self.w = []
+        
+    def forward(self, x, w, b):
```

Comment:
Most of the issues(6 out of 10) come from this:https://docs.quantifiedcode.com/python-anti-patterns/correctness/method_could_be_a_function.html
Python raises the 'Method could be a function' error whenever you don't use self in functions inside class

Reply:
so can I remove self in the function?